spacy/pipeline/coordinationruler.py

```diff
@@ -0,0 +1,321 @@
+from typing import List, Callable, Optional, Union
+from pydantic import BaseModel, validator
+import re
+import en_core_web_sm
```

Comment:
We'll want to find another solution for this, because we don't want to enforce all users to have exactly this model in their environment